docs/tasks/administer-cluster/kube-router-network-policy.md

```diff
@@ -0,0 +1,25 @@
+---
+approvers:
+- murali-reddy
+title: Using Kube-router for NetworkPolicy
+---
+
+{% capture overview %}
+This page shows how to use [Kube-router](https://github.com/cloudnativelabs/kube-router) for NetworkPolicy.
+{% endcapture %}
+
+{% capture prerequisites %}
+* Install Kube-router add-on for Kubernetes.
+{% endcapture %}
+
+{% capture steps %}
+## Installing Kube-router addon
+You can deploy Kube-router add-on through cluster intstallers like Kops, kubeadm, bootkube etc. Please follow the [trying Kube-router with cluster installers](https://github.com/cloudnativelabs/kube-router/tree/master/Documentation#try-kube-router-with-cluster-installers) guide.
```

Comment:
Suggested edit: "You can deploy `the` Kube-router add-on through cluster intstallers like Kops, kubeadm, bootkube`,` etc."